.eslintrc.js

```diff
@@ -10,6 +10,10 @@ module.exports = {
     __VERSION__: true,
     CSS: true
   },
+  rules: {
+    'import/prefer-default-export': 'off',
+    'import/no-named-as-default': 'off'
```

Comment:
why not turn of the rule locally where needed?

Reply:
Ideally we should not have to configure eslintrc anywhere except of one central place for cssinjs org

Reply:
will fix it since it's quicly done